relay-pii/src/regexes.rs

```diff
@@ -106,186 +106,239 @@ macro_rules! ip {
     (v6s) => { "[0-9a-fA-F]{1,4}" };
 }
 
+macro_rules! regex {
+    ($name:ident, $rule:expr) => {
+        #[allow(non_snake_case)]
+        mod $name {
+            use super::*;
+            pub static $name: Lazy<Regex> = Lazy::new(|| Regex::new($rule).unwrap());
```

Comment:
Could take the opportunity and switch this to `std::sync::LazyLock` (or we do that separately).